index.html

```diff
@@ -1,13 +1,15 @@
 <!doctype html>
-<html>
+
+<html lang="en">
 <head>
-	<meta charset="utf-8">
-	<title>HTML & CSS Training</title>
-	<meta name="description" content="training">
-	<meta name="author" content="Wolox">
-	<link rel="stylesheet" type="text/css" href="application.css">
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" href="application.css" type="text/css">
 </head>
 <body>
-	<h1 class="title">Hello World</h1>
+    <h1 class="title"><img class="title-decoration" src="assets/triangle.svg" alt="">Front-End Tools</h1> 
```

Comment:
Indentation: Still a few extra spaces :P
This is not wrong, but you can do it better! Try to use only one html element, the `h1` is ok. And then add the image usign a pseudo-element, IE: `&::before`

Reply:
I didn't saw it, its done!

Reply:
>.<